contracts/DIDToken.sol

```diff
@@ -31,56 +31,40 @@ contract DIDToken is Approvable {
     owner = msg.sender;
     name = "Distense DID";
     symbol = "DID";
-    numDIDOutstanding = 0;
-    numContributors = 0;
+    DIDOutstanding = 123;
+    numContribs = 321;
     decimals = 1;
+    approveAddress(owner);
   }
 
   function updateProfile(string _email, bytes8 _countryCode) {
     contributors[msg.sender].email = _email;
     contributors[msg.sender].countryCode = _countryCode;
   }
 
-  function issueDIDReward(address _contribAddress, uint256 _amount, string _taskID) internal returns (bool) {
-    numDIDOutstanding = numDIDOutstanding.add(_amount);
+  function issueDIDReward(address _contribAddress, uint256 _amount, string _taskID) onlyApproved returns (bool) {
+    DIDOutstanding = DIDOutstanding.add(_amount);
```

Comment:
Wanted to shrink it but keep the DID explicitness.  Could change to `numOutStandingDID` or `outstandingDID` (I'd go for the latter).